__tests__/db.test.js

```diff
@@ -35,29 +36,87 @@ describe("App", () => {
           .get("/api/topics")
           .expect(200)
           .then(({ body }) => {
-            console.log(body.topics, " <<<< my bodyyy");
             body.topics.forEach((element) => {
               let objKeys = Object.keys(element);
-              expect(objKeys.includes("slug")).toBe(true);
-              expect(objKeys.includes("description")).toBe(true);
+              if (objKeys.length > 0) {
```

Comment:
This could potentially cause a false positive test, if this if statement never runs there isn't any assertions in this test